processor/test/integration/projects.test.ts

```diff
@@ -61,19 +62,39 @@ vi.mock('../../src/meili.js', () => {
   return { meiliIndex }
 })
 
-type GiteaDB = typeof giteaDBImported
+type GiteaDBMock = MockProxy<typeof giteaDBImported>
 
 vi.mock('../../src/giteaDB.js', () => {
-  const giteaDB: MockProxy<GiteaDB> = mock<GiteaDB>()
+  const giteaDB: GiteaDBMock = mock<typeof giteaDBImported>()
   giteaDB.waitForNonEmpty.mockReturnValue(Promise.resolve())
   giteaDB.waitForRepoMigration.mockReturnValue(Promise.resolve())
-  giteaDB.subscribeToRepoDeletions.mockReturnValue(
+  giteaDB.subscribeToRepoEvents.mockReturnValue(
     Promise.resolve({ unsubscribe: () => {} }),
   )
+  giteaDB.getAllRepoInfo.mockReturnValue(Promise.resolve([]))
+
   return giteaDB
 })
 
-const giteaDB = giteaDBImported as MockProxy<GiteaDB>
+async function dispatchRepoEvent(
+  giteaDB: GiteaDBMock,
+  repoInfo: giteaDBImported.RepoInfo,
+  event: ReplicationEvent,
+) {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  for (const [_, callback] of giteaDB.subscribeToRepoEvents.mock.calls) {
```

Comment:
Could you allow `_+` as unused vars in eslintrc please?

Reply:
Resolved in 9d9f69b0.